apps/docs/vite.config.ts

```diff
@@ -3,4 +3,7 @@ import react from '@vitejs/plugin-react';
 
 export default defineConfig({
   plugins: [react()],
+  optimizeDeps: {
```

Comment:
Interesting, I'm curious about this.

Reply:
Ah yeah I removed this in my recent push. Thanks for pointing that out. I was having local issues with storybook cache and was experimenting with vite's config file to fix the issue. I forgot I had included that in this PR. It doesn't do anything, so i removed it!